parser/comments_test.go

```diff
@@ -0,0 +1,1296 @@
+package parser
+
+import (
+	"fmt"
+	"github.com/robertkrimen/otto/ast"
+	"reflect"
+	"testing"
+)
+
+func checkComments(actual []*ast.Comment, expected []string, position ast.CommentPosition) error {
+	if len(actual) != len(expected) {
+		return fmt.Errorf("The number of comments is not correct. %v != %v", len(actual), len(expected))
```

Comment:
I have lower cased my errors, but I have not moved them. Yet. The errors used in tests, should they be moved? If I only use an error once, I don't think it should be moved.